presto-native-execution/README.md

```diff
@@ -54,9 +50,9 @@ Makefile targets exist for showing, fixing and checking formatting, license
 headers and clang-tidy warnings.  These targets are shortcuts for calling
 `./scripts/check.py`.  
 
-CircleCi runs `make format-check`, `make header-check` and `make tidy-check` as
-part of our continious integration.  Pull requests should pass format-check and
-header-check without errors before being accepted.  
+GitHub Actions run `make format-check`, `make header-check` and
+`make tidy-check` as part of our continuous integration.  Pull requests should
+pass format-check and header-check without errors before being accepted.  
```

Comment:
Should we also clarify that PRs must pass linux-build and other jobs before being accepted?

Reply:
This is paragraph about formatting, but we can mention that all jobs need to pass.

Reply:
Let's mention that all jobs to pass to make it clear.